app/Http/Controllers/Backend/GeoController.php

```diff
@@ -120,6 +119,6 @@ public static function calculateDistanceBetweenCoordinates(
         $distance = acos(sin($latA) * sin($latB) + cos($latA) * cos($latB) * cos($lonB - $lonA))
                     * $equatorialRadiusInKilometers;
 
-        return round($distance, $decimals);
+        return round($distance, 3) * 1000;
```

Comment:
Change `$equatorialRadiusInKilometers = 6378.137;` to  `$equatorialRadiusInMeters = 6378137;` and return `round($distance, 0)`